### PR TITLE
Feature/mob 2462 remote strings chat

### PR DESCRIPTION
**Pull request by Pelkar**
Implemented remote strings for chat, fixed multi argument string fetching

### CODE REVIEW

File: widgetssdk/src/main/java/com/glia/widgets/StringProviderImpl.kt [hunk 1]
```diff
@@ -6,8 +6,8 @@ class StringProviderImpl(private val resourceProvider: ResourceProvider): String
 
     override fun getRemoteString(stringKey: Int, vararg values: String): String {
         // TODO IMPLEMENT NEW METHOD FROM CORE
-        val fallback = resourceProvider.getString(stringKey, values)
         val key = resourceProvider.getResourceKey(stringKey)
+        val fallback = resourceProvider.getString(stringKey, *values)
```
Comment (AndriiHorishniiMOC) [inline widgetssdk/src/main/java/com/glia/widgets/StringProviderImpl.kt:10]: I don't get the idea of the fallback.

**Review by AndriiHorishniiMOC — APPROVED** [review]

File: widgetssdk/src/main/java/com/glia/widgets/StringProviderImpl.kt (hunk 1 above)
Comment (Pelkar) [reply widgetssdk/src/main/java/com/glia/widgets/StringProviderImpl.kt:10]: of fallback or of the change in the fallback?

Fallback is the native string value that will be used ONLY if no value is present in the remote strings

**Review by andrews-moc — APPROVED** [review]

File: widgetssdk/src/test/snapshots/images/com.glia.widgets.chat_ChatViewQuickReplySnapshotTest_withWithUnifiedTheme.png
Comment (AndriiHorishniiMOC) [inline widgetssdk/src/test/snapshots/images/com.glia.widgets.chat_ChatViewQuickReplySnapshotTest_withWithUnifiedTheme.png:1]: Initially, the gradient for the app bar title was vertical. Now, it is horizontal.

**Review by AndriiHorishniiMOC — CHANGES_REQUESTED** [review]
Looks like we have issues with Git LFS

File: .gitattributes [hunk 2]
```diff
@@ -1 +1,2 @@
 widgetssdk/src/test/snapshots/**/*.png filter=lfs diff=lfs merge=lfs -text
+*.png filter=lfs diff=lfs merge=lfs -text
```
Comment (AndriiHorishniiMOC) [inline .gitattributes:2]: Do not need to add all PNG files with LFS.

**Review by AndriiHorishniiMOC — APPROVED** [review]
Good work!

**Pull request merged**